go.mod

```diff
@@ -1,16 +1,18 @@
 module github.com/bufbuild/buf
 
-go 1.21.0
+go 1.22.0
```

Comment:
I thought we needed to support the latest three versions of Go, not just latest two, for customers that build from source? Does this change here mean that we have changed our policy?

I actually noticed the other week that the Google Cloud Go client libraries are also changing to only support two latest versions, [but not starting until January](https://github.com/googleapis/google-cloud-go?tab=readme-ov-file#go-versions-supported). Maybe we are following suit?